nodes/nomos-node/src/main.rs

```diff
@@ -4,19 +4,16 @@ use nomos_node::{
     OverlayArgs, Tx,
 };
 
-mod bridges;
-
 use clap::Parser;
 use color_eyre::eyre::{eyre, Result};
 use nomos_core::{
     da::{blob, certificate},
     tx::Transaction,
 };
-use nomos_http::bridge::{HttpBridge, HttpBridgeSettings};
-use nomos_mempool::network::adapters::libp2p::{Libp2pAdapter, Settings as AdapterSettings};
-use nomos_network::backends::libp2p::Libp2p;
+
+use nomos_mempool::network::adapters::libp2p::Settings as AdapterSettings;
```

Comment:
Why is this change here? This is related to mempool right?

Reply:
Yes, we need this setting to construct a mempool service.

Reply:
But why the adapter and the backend is gone now?

Reply:
In the previous, we had type aliases for creating bridges.